logicmonitor/sensors/logicmonitor_sensor.py

```diff
@@ -0,0 +1,102 @@
+import json
+import requests
+import sensor_constants as C
+from flask import request, Flask
+from st2reactor.sensor.base import Sensor
+
+
+class LogicMonitorSensor(Sensor):
+    def __init__(self, sensor_service, config=None):
+        self._config = config
+        self._sensor_service = sensor_service
+
+        self._host = "0.0.0.0"
+        self._port = "5000"
+
+        if config != None:
+            self._auth_enabled = config.get("auth_enabled")
+
+        self._app = Flask(__name__)
+        self._log = self._sensor_service.get_logger(__name__)
+
+    def setup(self):
+        pass
+
+    def run(self):
+        @self._app.route("/", methods=["POST"])
+        def handle_webhook():
+
+            # Get JSON data from POST request
+            try:
+                data = json.loads(request.data)
+                self._log.info(C.LOG_LOADING_DATA)
+            except Exception:
+                self._log.info(C.LOG_FAILED_TO_PARSE_JSON)
+                return C.RES_FAILED_TO_PARSE_JSON
+
+            # Get API Key from POST request
+            try:
+                _apiKey = data[C.API_KEY_KEY]
+                self._log.info(C.LOG_API_KEY_EXISTS)
+            except Exception:
+                self._log.info(C.LOG_API_KEY_DOES_NOT_EXIST)
+                return C.RES_API_KEY_DOES_NOT_EXIST
+
+            if self._auth_enabled is True:
```

Comment:
I suggest simplifying your condition. `is <bool>` is rather unusual in python.
```suggestion
            if self._auth_enabled:
```

Reply:
This was a change requested during a code review on our side, so, will leave it as is.